Automattic/woocommerce-payments
# Transaction Details → Dispute Details – show dispute amount in store/dispute currency

Part of #7124

> **Note**
> This PR is behind feature flag `_wcpay_feature_dispute_on_transaction_page`

#### Changes proposed in this Pull Request
This PR changes the Transaction Details → dispute amount to display in the store's currency rather than the charge currency, matching the currency of the headline transaction amount (changed in #7159).

For inquiries (dispute status `warning_needs_response`), the dispute amount will be rendered in the charge currency due to limitations with the dispute API – since no dispute balance transaction has been deducted, the amount is provided only in the charge currency.

[See design](https://www.figma.com/file/RiEQmaKRI7u9USAI3lyZz0VX/Dispute-Flows?node-id=6988:122148&mode=dev)

**Dispute** amount rendered in store currency

![dispute](https://github.com/Automattic/woocommerce-payments/assets/3147296/5d7a3d84-515b-4d01-b35f-cffd6092db9a)

**Inquiry** dispute amount rendered in charge currency

![inquiry](https://github.com/Automattic/woocommerce-payments/assets/3147296/44083ef0-ea36-4c55-aa8d-dae352c2e8a3)

#### Testing instructions






1. Enable feature flag by running `update_option( '_wcpay_feature_dispute_on_transaction_page', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/) or by modifying your database table `wp_options` directly.
1. Enable **Payments → Settings → Multi-currency**.
3. Visit **Payments → Overview** and note your store's deposit currency or currencies, visible as balance tabs.
4. Enable a non-store currency in **WooCommerce → Settings → Multi-currency**. Enable the currency switcher if using the Storefront theme (otherwise, show the front-end currency switcher via the block editor or another method). 
6. Create a **disputed transaction in (one of) the store's currency** using the card `4000000000000259` at checkout. The transaction details screen should show the dispute amount in the store's currency, ~matching the headline transaction currency~.
8. Create a **disputed transaction in a non-store currency** using the currency switcher and the card `4000000000000259` at checkout. The transaction details screen should show the dispute amount in the store's currency, ~matching the headline transaction currency~.
8. Create a **disputed inquiry in a non-store currency** using the currency switcher and the card `4000000000001976` at checkout. The transaction details screen should show the dispute amount in the transaction currency.

**Update to test instructions**
* I've changes the base to `develop`, therefore the transaction headline amount will not display in the store's currency until #7159 is merged.

-------------------


**Post merge**
